packages/smock/tsconfig.json

```diff
@@ -1,6 +1,4 @@
 {
-  "extends": "../../tsconfig.json",
-  "files": [
-      "./hardhat.config.ts"
-  ]
+  "extends": "./tsconfig.build.json",
+  "include": ["./src", "./test"]
```

Comment:
I'm not super familiar with the nuances of tsconfig, but do you know why this had to change?

cc @smartcontracts

Reply:
Let me play with this again to confirm that this was needed.

Reply:
Ok so I think what was happening was that `eslint` was not linting directories or files with the relevant extension if a `files` key-value pair was specified in the `tsconfig.json`. If it was specified and had a value, `eslint` would also lint the directories specified by the `include` key-value pair. 

I think the cleanest way to about fixing this is to remove the `files` key-value pair from relevant `tsconfig.json`s (I've gone ahead and done this in the most recent commit - the linter seemed to be happy with it).

That being said, I'm not sure if there was a specific reason that the `files` key-value pair was originally specified, so if we do need to keep it, we might be able to use this previous workaround OR I can look for an alternative.